_posts/en/newsletters/2020-10-14-newsletter.md

```diff
@@ -123,12 +123,63 @@ changes to popular Bitcoin infrastructure software.
 meeting, highlighting some of the important questions and answers.  Click on a
 question below to see a summary of the answer from the meeting.*
 
-FIXME:jnewbery or jonatack
+[BIP-325: Signet][review club #18267] is a PR ([#18267][Bitcoin Core #18267]) by
+Kalle Alm that implements a new kind of Bitcoin test network. Signet has since
```

Comment:
should we link "Signet" to https://bitcoinops.org/en/topics/signet/?

Reply:
I'll make this change, but to answer your question: in general, the first time we mention a topic in an independent section, we should link to it (exception: we've already mentioned that topic a lot in previous sections so the reader should already be quite familiar with it).

Reply:
I've changed the introduction sentence slightly and linked to the signet topic from there.